src/Bongo/BongoClient.cs

```diff
@@ -3,113 +3,60 @@
 using System.Linq;
 using System.Net;
 using System.Threading.Tasks;
-using Bongo.Impala;
+using Akka.Actor;
+using Akka.Routing;
+using Bongo.Actors.BulkOperations;
+using Bongo.Actors.BulkOperations.Messages;
+using Bongo.Actors.Pools;
+using Bongo.Actors.Query;
+using Bongo.Actors.Query.Messages;
+using Bongo.Actors.Tables;
 
 namespace Bongo
 {
     public class BongoClient : IDisposable
     {
-        private readonly TimeSpan _timeout;
-        private readonly ImpalaClient _impalaClient;
+        private readonly ActorSystem _system;
+        private readonly IActorRef _bulkManager;
+        private readonly IActorRef _queryManager;
 
-        public BongoClient(IPEndPoint host, TimeSpan? timeout = null)
+        public BongoClient(List<IPEndPoint> impalaHosts, int hostPoolSize)
```

Comment:
I think this client would be more dev-friendly if it accepted a string of comma-separated URIs/host[:port] options. (Like something you'd probably specify in a config file.)

```csharp
const int DefaultPort = 21000;
IEnumerable<IPEndPoint> ParseHostNames(string hostNames)
{
	return Regex.Split(hostNames, @"\s*,\s*")
		.Select(h => h.Contains("://") ? h : $"tcp://{h}")
		.Select(h => new Uri(h))
		.Select(uri => uri.IsDefaultPort ? new Uri($"{uri.Scheme}://{uri.Host}:{DefaultPort}") : uri)
		.SelectMany(uri => Dns.GetHostAddresses(uri.DnsSafeHost)
			.Select(ip => new IPEndPoint(ip, uri.Port)));
}
```

Usage:
```csharp
ParseHostNames("dev-core.syn.am:21000,127.0.1.2:12345, syncromatics.engineering")
```

Produces an enumerable of `IPEndPoint`s for:
* 127.0.0.1:21000
* 127.0.1.2:12345
* 192.30.252.153:21000
* 192.30.252.154:21000

(The last two are because syncromatics.engineering has an A record with two addresses ;) )